crates/consensus/src/transaction/mod.rs

```diff
@@ -100,18 +99,6 @@ pub trait SignableTransaction<Signature>: Transaction {
     fn into_signed(self, signature: Signature) -> Signed<Self, Signature>
     where
         Self: Sized;
-
-    /// Encode with a signature. This encoding is usually RLP, but may be
-    /// different for future EIP-2718 transaction types.
-    fn encode_signed(&self, signature: &Signature, out: &mut dyn BufMut);
-
-    /// Decode a signed transaction. This decoding is usually RLP, but may be
-    /// different for future EIP-2718 transaction types.
-    ///
-    /// This MUST be the inverse of [`SignableTransaction::encode_signed`].
-    fn decode_signed(buf: &mut &[u8]) -> alloy_rlp::Result<Signed<Self, Signature>>
```

Comment:
why was this removed? it's useful for converting from mempool rlp -> typed tx

Reply:
i see in the pr description that this is possible through TxEnvelope, could you give me a small example? we need to note this down for network crate authors, since it is likely a very important feature